packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -14,6 +16,23 @@ const NominationBanner = ({ nomination }) => {
   const time = newDate.toLocaleDateString();
   const minutes = newDate.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
   const finalDate = `${time} – ${minutes}`;
+  const [activeNomination, setActiveNomination] = useContext(ActiveNominationContext);
+  const declineStatus = 'Declined'
+
+  function declineApplication() {
```

Comment:
Do you have an easy way of providing a screen recording that demonstrates this functionality (e.g. clicking the "Decline Application" button and verifying that the email gets sent)?

I'm having some issues validating this locally, and a video recording would save me a ton of troubleshooting time.

Reply:
https://user-images.githubusercontent.com/50112365/122856694-d5690600-d2dc-11eb-9a2d-8e71e1c75d65.mov

Quicktime for the win.

Right now the status bar turns grey, but I believe product is writing a story to turn that red, and then turn the 'Decline Application' button to 'Declined'